components/project.py

```diff
@@ -25,3 +25,36 @@ class LogWidget(BaseElement):
 
     # Group Locators
     log_items = GroupLocator(By.CSS_SELECTOR, '#logFeed .db-activity-item')
+
+
+class CreateRegistrationModal(BaseElement):
+    modal_window = Locator(By.CSS_SELECTOR, '[data-test-new-registration-modal]')
+    cancel_button = Locator(
+        By.CSS_SELECTOR, 'button[data-test-new-registration-modal-cancel-button]'
+    )
+    create_draft_button = Locator(
+        By.CSS_SELECTOR, 'button[data-test-new-registration-modal-create-draft-button]'
+    )
+
+    # Group Locators
+    schema_list = GroupLocator(
+        By.CSS_SELECTOR, '[data-test-new-registration-modal-schema]'
+    )
+
+    def get_schema_names_list(self):
+        """Returns the schema names from the schema list"""
+        names_list = []
+        for schema in self.schema_list:
+            names_list.append(schema.text)
+        return names_list
+
+    def select_schema_radio_button(self, schema_name='Open-Ended Registration'):
+        """Selects the radio button corresponding to the given schema name"""
+        for schema in self.schema_list:
+            if schema.text == schema_name:
+                schema.find_element_by_css_selector('.ember-view').click()
```

Comment:
Unless you're expecting this `if` to succeed multiple times during the loop, add a `break` statement after this to exit the loop.

Reply:
Oops. My Bad. Fixed!